src/xenia/app/premake5.lua

```diff
@@ -71,17 +72,24 @@ project("xenia-app")
       "xcb",
       "X11-xcb",
       "vulkan",
+      "SDL2",
     })
 
   filter("platforms:Windows")
     links({
+      "delayimp", -- Enable dll delayed loading for msvc
```

Comment:
Could you change the comment to something like "This library implements delayed loading on Windows" instead?
The current comment makes me think this is a linker option, which it isn't.

Reply:
Changed this and split linking related changes shared with #1493 into an extra commit.